test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -127,6 +127,22 @@ function (_, $, Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthFo
     });
   }
 
+  function setupAdditionalAuthButton() {
```

Comment:
General: Since we have e2e tests, might be nice to put a new one in there as well.